_posts/0000-01-02-Endonerios.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![image](https://user-images.githubusercontent.com/64779454/80960889-744d4d00-8e12-11ea-80b0-94f472dd938d.png)

Reply:
this is line 5 bot wtf noob

Reply:
up cour

!!!!!!!!!!!!!!!!!!!!111111111111111111

Reply:
shut up